backend/config/redis.go

```diff
@@ -0,0 +1,41 @@
+package config
+
+import (
+	"errors"
+
+	m "github.com/garrettladley/mattress"
+)
+
+type RedisSettings struct {
+	Username string            `yaml:"username"`
+	Host     string            `yaml:"host"`
+	Port     uint              `yaml:"port"`
+	Password *m.Secret[string] `yaml:"password"`
+	DB       int               `yaml:"db"`
+	// TLSConfig *TLSConfig       `yaml:"tlsconfig"`
```

Comment:
is this needed for prod? should we open an issue so we remember to uncomment/enable this?